components/emailable/emailable.app.mjs

```diff
@@ -1,11 +1,58 @@
+import { axios } from "@pipedream/platform";
+
 export default {
   type: "app",
   app: "emailable",
-  propDefinitions: {},
+  propDefinitions: {
+    email: {
+      type: "string",
+      label: "Email",
+      description: "The email you want to verify",
+    },
+    emails: {
+      type: "string[]",
+      label: "Emails",
+      description: "A list of emails to verify in batch (up to 50,000)",
+    },
+    url: {
+      type: "string",
+      label: "URL",
+      description: "A URL that will receive the batch results via HTTP POST",
+      optional: true,
+    },
+  },
   methods: {
-    // this.$auth contains connected account data
-    authKeys() {
-      console.log(Object.keys(this.$auth));
+    getUrl(path) {
+      return `https://api.emailable.com/v1${path}`;
+    },
+    getAuthParams(data) {
+      return {
+        api_key: this.$auth.api_key,
```

Comment:
```suggestion
        api_key: this._apiKey(),
```

Reply:
Is just to keep the pattern, do the same of the others apps, but will approve and mention here totake the decision.

@malexanderlim @danhsiung